functions/tests/test_stripe_checkout.py

```
@@ -0,0 +1,727 @@
+"""
+Comprehensive end-to-end tests for Stripe checkout functionality.
+Tests the complete checkout session creation workflow including validation,
+pricing calculations, and error handling scenarios.
+"""
+
+import json
+import math
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)